doc/markdown/user/implementation/mapping/CUDA.md

```diff
@@ -29,10 +29,11 @@ NOTE: You have to be careful when mixing alpaka and non alpaka CUDA code. The CU
 |CUDA|alpaka|
 |---|---|
 |\_\_host\_\_|ALPAKA_FN_HOST|
-|\_\_global\_\_|ALPAKA_FN_HOST_ACC|
-|\_\_device\_\_ \_\_host\_\_|ALPAKA_FN_HOST_ACC|
-|\_\_device\_\_|ALPAKA_FN_ACC_CUDA_ONLY|
+|\_\_device\_\_|ALPAKA_FN_ACC*|
```

Comment:
I am confused, both lines are `ALPAKA_FN_ACC*` ...

Reply:
Ok, but let's write a comment below to describe that.
Here, we should also write which ones are "internal" for accelerator backends and which ones are "external" for users.